package.json

```diff
@@ -51,5 +51,8 @@
   "resolutions": {
     "selenium-webdriver": "4.17.0"
   },
-  "packageManager": "yarn@3.5.0"
+  "packageManager": "yarn@3.5.0",
+  "dependencies": {
+    "leaflet": "^1.9.4"
```

Comment:
Just a thought, would dependabot pick up any leaflet upgrades and auto merge them? Do we need to keep an eye on that?